CODEOWNERS

```diff
@@ -0,0 +1,88 @@
+
+# Builder
+/code/builders/builder-manager/  @ndelangen @valentinpalkovic
+/code/builders/builder-vite/     @ndelangen @valentinpalkovic @IanVS
+/code/builders/builder-webpack5/ @ndelangen @valentinpalkovic
+
+# Frameworks
+/code/frameworks/angular/                 @valentinpalkovic @yannbf
+/code/frameworks/html-vite/               @kasperpeulen @kylegach
+/code/frameworks/html-webpack5/           @kasperpeulen @kylegach
+/code/frameworks/nextjs/                  @valentinpalkovic @kasperpeulen @yannbf
+/code/frameworks/react-vite/              @valentinpalkovic @kasperpeulen
+/code/frameworks/react-webpack5/          @valentinpalkovic @kasperpeulen
+/code/frameworks/svelte-vite/             @kasperpeulen @JReinhold
+/code/frameworks/svelte-webpack5/         @kasperpeulen @JReinhold
+/code/frameworks/sveltekit/               @kasperpeulen @JReinhold
+/code/frameworks/vue3-vite/               @kasperpeulen @yannbf @JReinhold
+/code/frameworks/vue3-webpack5/           @kasperpeulen @yannbf @JReinhold
+/code/frameworks/web-components-vite/     @kasperpeulen @JReinhold
+/code/frameworks/web-components-webpack5/ @kasperpeulen @JReinhold
+
+# Lib
+/code/lib/channels/       @ndelangen @kasperpeulen
+/code/lib/cli/            @yannbf @valentinpalkovic @ndelangen
+/code/lib/cli-sb/         @yannbf @valentinpalkovic @ndelangen
+/code/lib/cli-storybook/  @yannbf @valentinpalkovic @ndelangen
+/code/lib/client-logger/  @ndelangen @yannbf
+/code/lib/codemod/        @kasperpeulen @ndelangen
+/code/lib/core-common/    @ndelangen @yannbf
+/code/lib/core-events/    @ndelangen @kasperpeulen
+/code/lib/core-server/    @ndelangen @JReinhold
+/code/lib/core-webpack/   @valentinpalkovic @ndelangen
+/code/lib/csf-plugin/     @ndelangen @valentinpalkovic
+/code/lib/csf-tools/      @kasperpeulen @shilman
+/code/lib/docs-tools/     @JReinhold @shilman
+/code/lib/instrumenter/   @yannbf @kasperpeulen
+/code/lib/manager-api/    @ndelangen @valentinpalkovic @kasperpeulen
+/code/lib/node-logger/    @yannbf @ndelangen
+/code/lib/preview/        @ndelangen @kasperpeulen
+/code/lib/preview-api/    @yannbf @ndelangen
+/code/lib/react-dom-shim/ @ndelangen @valentinpalkovic
+/code/lib/router/         @ndelangen @JReinhold
+/code/lib/telemetry/      @shilman @yannbf @ndelangen
+/code/lib/theming/        @cdedreuille @ndelangen @JReinhold
+/code/lib/types/          @kasperpeulen @ndelangen
+
+# Presets
+/code/presets/create-react-app/ @valentinpalkovic @ndelangen
+
+# Renderers
+/code/renderers/html/           @kasperpeulen @JReinhold
+/code/renderers/react/          @valentinpalkovic @kasperpeulen
+/code/renderers/server/         @shilman @valentinpalkovic
+/code/renderers/svelte/         @JReinhold @kasperpeulen
+/code/renderers/vue3/           @kasperpeulen @JReinhold
+/code/renderers/web-components/ @kasperpeulen @JReinhold
+
+# UI
+/code/ui/blocks/     @JReinhold @cdedreuille
+/code/ui/components/ @cdedreuille @JReinhold
+/code/ui/manager/    @ndelangen @JReinhold @cdedreuille
```

Comment:
```suggestion
# UI
/code/ui/.storybook/ @JReinhold @cdedreuille
/code/ui/blocks/     @JReinhold @cdedreuille
/code/ui/components/ @cdedreuille @JReinhold
/code/ui/manager/    @ndelangen @JReinhold @cdedreuille
```

Reply:
I think I should be added to `/code/ui/manager` too there are enough bits of that in the APIs I am responsible for.